### PR TITLE
fixed links in reports

### PR DESCRIPTION
**Pull request by joezsweet**
- changed label according to title links 
- fixed links in reports

### CODE REVIEW

File: title_links/routes.py [hunk 1]
```diff
@@ -5,137 +5,153 @@
...
-				limit_page_length=page_len,
-				order_by="if(_relevance, _relevance, 99999), idx desc, modified desc".format(doctype),
-				ignore_permissions = True if doctype == "DocType" else False, # for dynamic links
-				as_list=not as_dict)
-
-			# remove _relevance from results
-			frappe.response["values"] = [r[:-1] for r in values]
+                  page_len=10, filters=None, as_dict=False):
+
+    print "*************************** "
```
Comment (MaxMorais) [inline title_links/routes.py]: @joezsweet it's really necessary?

File: title_links/routes.py [hunk 2]
```diff
@@ -5,137 +5,153 @@
...
+            fields.append("""locate("{_txt}", `tab{doctype}`.`name`) as `_relevance`""".format(
+                _txt=frappe.db.escape((txt or "").replace("%", "")), doctype=frappe.db.escape(doctype)))
+
+            # In order_by, `idx` gets second priority, because it stores link count
+            from frappe.model.db_query import get_order_by
+            order_by_based_on_meta = get_order_by(doctype, meta)
+            order_by = "if(_relevance, _relevance, 99999), idx desc, {0}".format(order_by_based_on_meta)
+
+            print "***** "
+            print fields
```
Comment (MaxMorais) [inline title_links/routes.py]: and this?

**Comment by MaxMorais** [comment]
@joezsweet I'm approving your commit!

File: title_links/routes.py (hunk 2 above)
Comment (joezsweet) [inline title_links/routes.py]: I've removed ..in next commit .. 👍

**Pull request merged**